akka-projection-core/src/main/scala/akka/projection/HandlerRecovery.scala

```diff
@@ -9,20 +9,9 @@ import scala.concurrent.duration.FiniteDuration
 import akka.annotation.InternalApi
 import akka.util.JavaDurationConverters._
 
-trait HandlerRecovery[Envelope] {
```

Comment:
Now that this trait is gone, we can/should rename the file to `HandlerRecoveryStrategy.scala`

Reply:
:+1: